lang/java/avro/src/main/java/org/apache/avro/file/DataFileReader.java

```diff
@@ -58,7 +58,7 @@
     // read magic header
     byte[] magic = new byte[MAGIC.length];
     in.seek(0);
-    for (int c = 0; c < magic.length; c += in.read(magic, c, magic.length - c)) {
+    for (int c = 0; c < magic.length; c = in.read(magic, c, magic.length - c)) {
```

Comment:
is this change intended?

Reply:
It looks like this change doesn't belong to any commits from this PR.  An artifact from Github from changes on master since this PR was proposed?

I played with the "Resolve conflicts" directly on github for the first time, and it made a commit directly to your master branch -- my apologies, I'm not sure what I was expecting!